pkg/appconsts/initial_consts.go

```diff
@@ -2,20 +2,19 @@ package appconsts
 
 import (
 	"time"
-
-	"github.com/celestiaorg/go-square/v2/share"
 )
 
 // The following defaults correspond to initial parameters of the network that can be changed, not via app versions
 // but other means such as on-chain governance, or the node's local config
 const (
+	mebibyte = 1_048_576 // bytes
 	// DefaultGovMaxSquareSize is the default value for the governance modifiable
 	// max square size.
 	DefaultGovMaxSquareSize = 256
 
 	// DefaultMaxBytes is the default value for the governance modifiable
 	// maximum number of bytes allowed in a valid block.
-	DefaultMaxBytes = DefaultGovMaxSquareSize * DefaultGovMaxSquareSize * share.ContinuationSparseShareContentSize
+	DefaultMaxBytes = 32 * mebibyte
```

Comment:
The previous value was `31588352` and not 32mib, is this something we want to bump?

Reply:
Yes because the previous value isn't easy to remember or communicate. `31588352` is really specific but AFAIK these constants don't have to be that specific or dependent on `share.ContinuationSparseShareContentSize`

Reply:
FWIW this value was modified via governance to 8 mebibyte and it hasn't been an issue.